packages/oscal-react-library/src/components/OSCALProperties.tsx

```diff
@@ -0,0 +1,183 @@
+  /**
+   * Contains property elements
+   */
+  property: Property;
+}
+
+const OSCALProperty: React.FC<OSCALPropertyProps> = ({ property }) => {
+  const NO_INFORMATION = <NotSpecifiedTypography>Not Specified</NotSpecifiedTypography>;
+
+  return (
+    <StyledTooltip title={property.remarks ?? ""}>
+      <StyledTableRow key={property.uuid}>
+        <TableCell>{property.name ?? NO_INFORMATION}</TableCell>
+        <TableCell>{property.class ?? NO_INFORMATION}</TableCell>
+        <TableCell>{property.value ?? NO_INFORMATION}</TableCell>
+      </StyledTableRow>
+    </StyledTooltip>
+  );
+};
+
+interface OSCALPropertiesProps {
+  /**
+   * Contains a list of properties
+   */
+  properties?: Property[];
+  /**
+   * A namespace used for the title of a table
+   */
+  namespace?: string;
+}
+
+const OSCALProperties: React.FC<OSCALPropertiesProps> = ({ properties, namespace }) => {
+  return (
+    <>
+      <OSCALSystemImplementationTableTitle variant="h6" id={`${namespace}-table-title`}>
+        {namespace}
+      </OSCALSystemImplementationTableTitle>
+      <TableContainer sx={{ maxHeight: "20em" }}>
+        <Table aria-label="Components" sx={{ height: "max-content" }}>
+          <StyledTableHead>
+            <TableRow>
+              <StyledHeaderTableCell>Name</StyledHeaderTableCell>
+              <StyledHeaderTableCell>Class</StyledHeaderTableCell>
+              <StyledHeaderTableCell>Value</StyledHeaderTableCell>
+            </TableRow>
+          </StyledTableHead>
+          <TableBody>
+            {properties
+              ?.filter((property) => namespaceOf(property?.ns) === namespace)
+              .map((property) => (
+                <OSCALProperty
+                  property={property}
+                  key={`${namespaceOf(property?.ns)}-properties`}
+                />
+              ))}
+          </TableBody>
+        </Table>
+      </TableContainer>
+    </>
+  );
+};
+
+interface OSCALPropertiesDialogProps {
+  /**
+   * Contains properties elements
+   */
+  properties?: Property[];
+  /**
+   * Title of the element the properties resides under
+   */
+  title?: string | ReactElement;
+}
+
+export const OSCALPropertiesDialog: React.FC<OSCALPropertiesDialogProps> = ({
+  properties,
+  title,
+}) => {
+  const [open, setOpen] = React.useState(false);
+
+  if (!properties) {
+    return null;
+  }
+  const namespaceToProps: Record<string, Property[]> = {};
+  for (const prop of properties) {
+    const ns = namespaceOf(prop.ns);
+    namespaceToProps[ns] ??= [];
+    namespaceToProps[ns].push(prop);
+  }
+  const { [NIST_DEFAULT_NAMESPACE]: nist, ...thirdParties } = namespaceToProps;
+
+  const handleOpen = () => {
+    setOpen(true);
+  };
+  const handleClose = () => {
+    setOpen(false);
+  };
+
+  return (
+    <>
+      <OSCALPropertiesButton
+        size="small"
+        variant="outlined"
+        onClick={handleOpen}
+        aria-label={`${title} properties button`}
+        startIcon={<HomeIcon />}
```

Comment:
Instead of `Home` let's do https://mui.com/material-ui/material-icons/?selected=Construction

And then we can set 
```
title="Open Properties"
```

or something like that so that we can get _something_ on hover?

Reply:
I much more favor that icon for the button instead. I was thinking more real estate in terms of property haha

`title` holds the control name, resource title, or name of document (for metadata) currently. I think this is useful for the modal to contain this info at the very least. Are you suggesting the button have the title "Open Properties"?

Reply:
I am asking that the Button's `title` and `aria-label` attributes be set to "Open Properties"